python/setup.cfg

```diff
@@ -33,6 +33,7 @@ license_files =
 classifiers =
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
+    Programming Language :: Python :: 3.6
```

Comment:
QQ, wondering if we should support 3.6? Per recent sync meeting, the plan is to support just `py37,py38,py39` or higher versions.

Reply:
I would vote that we don't support 3.6 unless someone really feels we should. Looking at [endoflife.date/python](https://endoflife.date/python) shows that python 3.6 security support ends in a little over a month.

Reply:
I think that we discussed this in one of the python syncs and we agreed that supporting 3.6 was a bad idea.

Reply:
Reverted changes and dropped 3.6 support.